features/steps/steps.py

```diff
@@ -98,6 +149,19 @@ class representation_type_error:
     def __str__(self):
         return f"On instance {fmt(self.inst)} the {self.representation_id} shape representation does not have {self.representation_type} as RepresentationType"
 
+
+@dataclass
+class polyobject_structure_error:
+    inst: ifcopenshell.entity_instance
+    points: list
+    error_cause: str
+
+    def __str__(self):
+        if self.error_cause == 'duplicate_points':
+            return f"On instance {fmt(self.inst)} some of the points {self.points} are duplicate"
```

Comment:
Here I think @Ghesselink put in more effort in similar error categories (duplicate representation ids I think) to really also identify to the user, which are the duplicates. In a long polyline with precision I think it can be helpful. It's not a very important point for me. Can also be done later if/when a user requests.

Reply:
FYI. It's in the GEM003/GRF001 subbranch in the 'Then' statement 'The values must be {identical_or_unique}'.